buf.yaml

```diff
@@ -10,6 +10,8 @@ lint:
     - UNARY_RPC
     - COMMENT_FIELD
     - PACKAGE_DIRECTORY_MATCH
+    - ENUM_ZERO_VALUE_SUFFIX
+    - ENUM_VALUE_PREFIX
```

Comment:
I'd like good justifications for ignoring any more proto linters. Also I don't actually see any enum's here

Reply:
There are enums on `types/` and `23-commitment/`. The enum values need to match the IBC spec. That's why I ignored the prefix and suffix. Do you know if there's another way that I can ignore the lint errors?

Reply:
Ah the diff wasn't loading sorry.

Would using `KEY_ENCODING_URL` break the spec?

We may need to disable `ENUM_ZERO_VALUE_SUFFIX` anyway.

Reply:
updated